<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Station Editor</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;This Dialog is for Add a new Site or Edit a station&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Buses not in station:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Buses in station:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>671</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Station Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>95</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addall">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert All&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_remove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_removeall">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;Remove All</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_added">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>271</width>
     <height>401</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_hidden">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>90</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_branch">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>135</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Connected bus</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_branch">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>372</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>500</y>
     <width>105</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Station Type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_StatType">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>500</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Hydropower Station</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ThermalPower Station</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
